<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Layers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="projects_cmb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sequence:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sequences_cmb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shot:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shots_cmb"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="copyLocal_chb">
      <property name="text">
       <string>Copy to Local</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_btn">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
